### PR TITLE
Add some translation suggestions

### PR DESCRIPTION
**Pull request by mattsemar**

### CODE REVIEW

File: src/BattleProtos.cs [hunk 1]
```diff
@@ -122,49 +122,49 @@ public static void AddProtos()
...
-            ProtoRegistry.RegisterString("彩蛋8描述", "Young Icarus, no matter how you come here, you have proved your smart and power. Even facing unknown risks, you still light up your stars. " +
-                "Now you can proudly say, \"I've seen things that you absolutely can't believe, I've seen wormholes established in galaxies, I've seen fission rays flickering in swarms. All these moments will eventually fade away with time, like tears disappear in the rain.\" " +
-                "Continue your journey now! No matter what difficulties you encounter, don't be afraid and face them with a smile! Remember, you are a brave Icarus!", 
+            ProtoRegistry.RegisterString("彩蛋8描述", "Young Icarus, no matter how you came here, you have proved your smarts and strength. Even facing unknown risks, you still light up the stars. " +
+                "Now you can proudly say, \"I've seen things that you absolutely can't believe, I've seen wormholes established in galaxies, I've seen fission rays flickering in swarms. All those moments will be lost in time, like tears in rain.\" " +
```
Comment (mattsemar) [inline src/BattleProtos.cs:166]: Here I assume you were [quoting Blade Runner](https://en.wikipedia.org/wiki/Tears_in_rain_monologue). If not then what you had was fine

File: src/BattleProtos.cs [hunk 2]
```diff
@@ -122,49 +122,49 @@ public static void AddProtos()
...
-                "The stellar station will create regular ripples in the space, making itself as a beacon in dark universe, can be easily captured and attacked by the insects. " +
-                "In other words, <color=#c2853d>they will randomly choose a stellar station, and attack its solar system by wormholes. Stars with more stellar stations will have higher possibility to be selected, and stars without any stellar stations won't be attacked. </color>" +
-                "Obviously, I can't rely on giving up stellar stations, as the progress will come slowly without transportations. Instead, I have to find a way to defense. Hope I still have time to think about 'The Answer to Life, the Universe, and Everything'.", 
+            ProtoRegistry.RegisterString("彩蛋1描述", "Seems like these enemies are a kind of space insect. They live in deep space, feed on the tides of power without vision. " +
+                "The interstellar station creates regular ripples in space, making itself a beacon in a dark universe that can easily be captured and attacked by the insects. " +
```
Comment (mattsemar) [inline src/BattleProtos.cs:126]: Here I assume you're referring [to an ILS](https://dyson-sphere-program.fandom.com/wiki/Interstellar_Logistics_Station)?

File: src/BattleProtos.cs [hunk 3]
```diff
@@ -122,49 +122,49 @@ public static void AddProtos()
...
+                "Another thing I found is, <color=#c2853d>the intensity of the attack on different stars is independent. The first attack on a new star will start from the lowest intensity...</color> It seems I don't need to worry about the safety of planets with only for miners now.", 
                 "热核导弹的杀伤力确实很大，但是发射也太慢了，我得研究研究怎么把热核导弹造成微缩版本塞进炮弹里面去，那不就既有杀伤力又有速度嘛。我可真是个天才！" +
                 "唉，你说我这天才为什么就不得伊卡拉姆妹妹的喜欢呢？我为她专门点亮了一片星系告白，但她居然说：“前天看到了小白兔，昨天是小鹿，今天是你。”这不是嘲讽我像个动物一样蠢吗。算了算了，不提了，女人只会影响我造戴森球的速度。\n" +
                 "另外我发现，<color=#c2853d>这些虫子攻击不同星系的强度是独立的啊，进攻全新星系的虫群似乎会重新从最低强度开始...</color> 看来暂时是不怎么需要担心矿星的安全性问题了。");
-            ProtoRegistry.RegisterString("彩蛋5描述", "Damn it, a bit of anti-matter leakage almost lifted my pipelines, which caused me a lot of time to repair them. " +
-                "Wait, they can destroy my pipelines, why not insects? Good idea! Wait for me, fxxk insects, taste the anit-matter and die!", 
+            ProtoRegistry.RegisterString("彩蛋5描述", "Damn it, a bit of antimatter leakage almost destroyed my pipelines, which took me a lot of time to repair. " +
```
Comment (mattsemar) [inline src/BattleProtos.cs:150]: In the game they don't hyphenate antimatter

File: src/BattleProtos.cs [hunk 4]
```diff
@@ -122,49 +122,49 @@ public static void AddProtos()
...
-                "Now you can proudly say, \"I've seen things that you absolutely can't believe, I've seen wormholes established in galaxies, I've seen fission rays flickering in swarms. All these moments will eventually fade away with time, like tears disappear in the rain.\" " +
-                "Continue your journey now! No matter what difficulties you encounter, don't be afraid and face them with a smile! Remember, you are a brave Icarus!", 
+            ProtoRegistry.RegisterString("彩蛋8描述", "Young Icarus, no matter how you came here, you have proved your smarts and strength. Even facing unknown risks, you still light up the stars. " +
+                "Now you can proudly say, \"I've seen things that you absolutely can't believe, I've seen wormholes established in galaxies, I've seen fission rays flickering in swarms. All those moments will be lost in time, like tears in rain.\" " +
+                "Continue your journey now! No matter what difficulties you encounter, don't be afraid and face them with a smile! Remember, you are the mighty Icarus!",
```
Comment (mattsemar) [inline src/BattleProtos.cs:167]: I really like the prose in the diaries -- it's very evocative and fun to read. I tried not to mess up the tone as I really enjoyed it when I was playing

**Comment by mattsemar** [comment]
Oh I just now noticed the 2.0 branch, I'll rebase against that

**Comment by ckcz123** [comment]
Thanks for the translation improvements!

**Pull request merged**